.github/workflows/test.yml

```
@@ -0,0 +1,32 @@
+name: test
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.4, 3.5, 3.6, 3.7, 3.8, 3.9-dev, pypy2, pypy3]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install -U --force-reinstall setuptools pip
+        pip install -U .[develop]
```

```suggestion
        pip install --upgrade --force-reinstall setuptools pip
        pip install --upgrade .[develop]
```
Pip has both a `--upgrade` and a `--user option`.  Let's make it clear to the reader which one is happening.